pkg/custompluginmonitor/plugin/plugin.go

```diff
@@ -183,7 +182,12 @@ func (p *Plugin) run(rule cpmtypes.CustomRule) (exitStatus cpmtypes.Status, outp
 				glog.Errorf("Error in cmd.Process check %q", rule.Path)
 				break
 			}
-			err := syscall.Kill(-cmd.Process.Pid, syscall.SIGKILL)
+
+			m.Lock()
+			killed = true
+			m.Unlock()
```

Comment:
Change `killed` to `timeout`?

It is a bit weird to mark `killed` before the actual kill.